api/bases/glance.openstack.org_glances.yaml

```diff
@@ -1026,6 +1032,30 @@ spec:
               preserveJobs:
                 default: false
                 type: boolean
+              quotas:
+                properties:
+                  imageCountTotal:
+                    default: 0
```

Comment:
Right now the `tempest` failure is unrelated (it's due to some networking issue on the `SNO` environment), but in general this default is set to `0` because we want the user to explicitly pass the values documented in [1]. 
By having them set to `0` by default allow us to **not** enable the feature if the `quotas` struct is not passed in the `Glance` CR (and this preserves backward compatibility with what we already have in place):

```golang
// IsQuotaEnabled - return true if one of the QuotaLimits values is set
func (instance Glance) IsQuotaEnabled() bool {
  return (instance.Spec.Quotas.ImageSizeTotal > 0 ||
      instance.Spec.Quotas.ImageCountTotal > 0 ||
      instance.Spec.Quotas.ImageStageTotal > 0 ||
      instance.Spec.Quotas.ImageCountUpload  > 0)
}
```

We can implement a more strictly check here (e.g., enable only if it matches w/ [1]), but I think we can document this part in the `docs/` section of the operator, and explain how users can enable the feature.
 
[1] https://docs.openstack.org/glance/latest/admin/quotas.html#configuring-glance-for-per-tenant-quotas

Reply:
+1